sample/CSharp/host.json

```diff
@@ -25,5 +25,9 @@
             "visibilityTimeout": "00:00:10",
             "maxDequeueCount": 3
         }
+    },
+    "extensionBundle": {
```

Comment:
Can you add a new test as this setting is optional? We need to keep a default test that does not configure bundles.

Reply:
There is a test below `ValidExtenstionConfig_DoesNotThrowException` that verifies the optional configuration scenario

Reply:
Removing the config in sample/host.json right now. Since sample tests without the code that uses extension bundling would only include the tests that are already there at the moment.